### PR TITLE
Update environment

### PR DESCRIPTION
**Pull request by scharlottej13**
Updating some of these packages since dask-expr was incorporated into dask dataframe. The only one I had some trouble with was dask-deltatable (see https://github.com/dask-contrib/dask-deltatable/issues/81)

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -17,8 +16,8 @@ dependencies:
   - s3fs
   - universal_pathlib <0.2.0
   - boto3
-  - deltalake=0.15.3
+  - deltalake >=0.18
   # - dask-deltatable
   - pip
   - pip:
-    - git+https://github.com/fjetter/dask-deltatable.git@dask_expr
+    - git+https://github.com/dask-contrib/dask-deltatable.git@7706c22aa07f978f481ca4996dce8ca0121c812e
```
Comment (jrbourbeau) [inline environment.yml:23]: Looks like `dask-deltatable==0.3.3` should fix the issue you ran into? https://github.com/dask-contrib/dask-deltatable/issues/81

Comment (scharlottej13) [reply environment.yml:23]: yup that works!

Comment (scharlottej13) [inline environment.yml:23]:
```suggestion
    - dask-deltatable==0.3.3
```

File: environment.yml [hunk 2]
```diff
@@ -17,8 +16,8 @@ dependencies:
   - s3fs
   - universal_pathlib <0.2.0
   - boto3
-  - deltalake=0.15.3
+  - deltalake >=0.18
   # - dask-deltatable
```
Comment (jrbourbeau) [inline environment.yml:20]:
```suggestion
  - dask-deltatable=0.3.3
```

File: environment.yml [hunk 3]
```diff
@@ -17,8 +16,8 @@ dependencies:
...
   - pip
   - pip:
-    - git+https://github.com/fjetter/dask-deltatable.git@dask_expr
+    - dask-deltatable==0.3.3
```
Comment (jrbourbeau) [inline environment.yml:23]: Can remove this now

Comment (scharlottej13) [reply environment.yml:23]: I had to pip install it for things to work

Comment (jrbourbeau) [reply environment.yml:23]: Hmm that's strange but 🤷

**Pull request merged**